lib/puppet/defaults.rb

```diff
@@ -851,7 +851,8 @@ def self.default_vendormoduledir
       :mode => "0644",
       :owner => "service",
       :group => "service",
-      :desc => "Where individual hosts store and look for their certificate requests."
+      :deprecated  => :completely,
+      :desc => "This setting is deprecated."
```

Comment:
It would have been nice to change the default value to be `File.join(Puppet[:requestdir], "#{Puppet[:certname]}.pem")`, so that it just works. But we can't  change the default value for a setting in a .z release.

Reply:
Do we have a ticket so we can make this happen in a future release?

Reply:
Filed as https://tickets.puppetlabs.com/browse/PUP-9346